hack/docgen/api/gen-api-doc-config.json

```diff
@@ -31,5 +35,22 @@
         "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/": "Kubernetes api extensions ",
         "k8s.io/apimachinery/pkg/util/": "Kubernetes api utils "
     },
+    "packageGroups": [
+        {
+            "groupName": "cluster",
+            "packages": ["apps.kubeblocks.io", "workloads.kubeblocks.io"],
+            "order": 1
+        },
+        {
+            "groupName": "backup",
+            "packages": ["dataprotection.kubeblocks.io", "storage.kubeblocks.io"],
+            "order": 2
+        },
+        {
+            "groupName": "add-on",
+            "packages": ["workloads.kubeblocks.io"],
```

Comment:
"workloads.kubeblocks.io" is not an add-on package.

Reply:
fixed